dump/dump.go

```diff
@@ -444,3 +448,15 @@ func GetAllACLGroups(client *kong.Client, tags []string) ([]*kong.ACLGroup, erro
 	}
 	return aclGroups, nil
 }
+
+// ExcludeConsumersPlugins filter out consumer plugins
+func ExcludeConsumersPlugins(plugins []*kong.Plugin) []*kong.Plugin {
```

Comment:
Please don't export this method. Methods beginning with a capital letter are available outside the package. It makes sense to keep this method internal to the package only.